src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -516,6 +516,82 @@ default Path resolve(String other) {
         return resolve(getFileSystem().getPath(other));
     }
 
+    /**
+     * Resolves a path, or a sequence of paths against this path.
+     *
+     * <p> The result of this method is the same as would be obtained
+     * by resolving the first parameter path against this path, and then
+     * repeating the same procedure for each path in the sequence. Each
+     * single resolution in the chain behaves as specified by {@link
+     * #resolve(Path) resolve}.
+     *
+     * @implSpec
+     * The default implementation is equivalent for this path to:
+     * {@snippet lang=java :
+     * Path result = resolve(first);
+     * for (Path p : more) {
+     *     result = result.resolve(p);
+     * }
+     * }
+     *
+     * @param first the first path to resolve against this path
+     *
+     * @param more additional paths to be joined to form the path
```

Comment:
"additional paths to be joined to form the path" may be copied from elsewhere :-)

Reply:
Blindly, even.